### ISSUE TITLE
Create issues for calculators that the community can contribute

### ISSUE THREAD

**Issue by rahulporuri** [urbanemissions-info/pollutioncalculators#9]
In a personal conversation, there was mention of calculators that exist only in Excel files that don't have Python equivalents. If the community can contribute these calculators, it will be useful to create specific issues for each calculator and add pertinent information about the calculator.

It's not clear to me if #3 is one such example.

**Comment by d-saikrishna** [comment]
Yes #3 is one such example. I have edited it with all information. I'll open one such issue for each calculator.
